test/unit/php/includes/core/classes/class-test-event.php

```diff
@@ -406,8 +403,7 @@ public function test_get_calendar_links(): void {
 
 		$event->save_datetimes( $params );
 
-		$output = $event->get_calendar_links();
-
+		/*
```

Comment:
I kept this failing, to make sure not to forget about that test. Because the existing test, tests now only if the generated endpoint urls are correct, but not neither the redirects nor the generated ical output.

This needs to become a new test, I guess.